pythonforandroid/recipe.py

```diff
@@ -695,6 +701,7 @@ def build_arch(self, arch, *extra_args):
             shprint(
                 sh.ndk_build,
                 'V=1',
+                'NDK_DEBUG=' + ("1" if self.build_as_debuggable else "0"),
```

Comment:
I like the way you did because it's clear/explicit, but in theory you could directly cast to int.
Hence:
```python
'NDK_DEBUG=' + int(self.build_as_debuggable),
```
See:
```python
>>> int(True)
1
>>> int(False)
0
```

Reply:
hmm good point. I'll go with whatever you guys prefer, I personally like the explicit way though. unlike in other languages I think the python ternary operator is very neat and readable, while casting bool to int seems a bit arcane to me

Reply:
Yes let's keep it the explicit way :+1: